server/src/main/java/org/elasticsearch/action/search/TransportSearchHelper.java

```diff
@@ -30,13 +31,15 @@ static InternalScrollSearchRequest internalScrollSearchRequest(ShardSearchContex
     }
 
     static String buildScrollId(AtomicArray<? extends SearchPhaseResult> searchPhaseResults) {
-        try (RAMOutputStream out = new RAMOutputStream()) {
+        try {
+            // TODO replace with BytesStreamOutput to avoid endiannessreverser
+            ByteBuffersDataOutput out = new ByteBuffersDataOutput();
```

Comment:
Are we leaving this as a follow up?

Reply:
No, good catch, I had intended to update this one.  Done in 46fc8c1